### PR TITLE
[AIRFLOW-XXX] Reduction of the number of warnings in the documentation

### PR DESCRIPTION
**Pull request by mik-laj**
Hello,

I took today to correct the errors in the documentation. I limited their number considerably. I would ask you to accept this PR quickly so that I do not have to fight with conflicts.
I will try to correct the rest of the warnings in a separate PR, but I do not know when I will find the time.

Before:
````
build succeeded, 432 warnings.
````
After: 
````
build succeeded, 21 warnings.
````

Thanks in advance.

### CODE REVIEW

**Comment by feng-tao** [comment]
thanks @mik-laj. I think this needs to have a jira ticket as we should include it in next release.

**Comment by mik-laj** [comment]
Changes to the documentation do not need a ticket. I do not understand why this change in the documentation requires it.

File: docs/kubernetes.rst [hunk 1]
```diff
@@ -131,6 +131,10 @@ Kubernetes Operator
 
 
 .. autoclass:: airflow.contrib.operators.kubernetes_pod_operator.KubernetesPodOperator
+    :noindex:
```
Comment (ashb) [inline docs/kubernetes.rst:134]: What change does this make?

**Comment by ashb** [comment]
@mik-laj The reason this would (normally) need a jira ticket is that we commit to the master branch, and the Jira ticket is what what we use to track what we have-or-haven't yet cherry picked on to the release branch.

**Comment by ashb** [comment]
That said I'm taking a look at it now, and since I happen to be preparing the release for 1.10.3 I may be able to cherry-pick it directly.

File: docs/kubernetes.rst (hunk 1 above)
Comment (mik-laj) [reply docs/kubernetes.rst:134]: This is related to this warning:
````
/workspace/airflow/contrib/operators/kubernetes_pod_operator.py:docstring of airflow.contrib.operators.kubernetes_pod_operator.KubernetesPodOperator:1: WARNING: duplicate object description of airflow.contrib.operators.kubernetes_pod_operator.KubernetesPodOperator, other instance in /workspace/docs/code.rst, use :noindex: for one of them
/workspace/airflow/contrib/operators/kubernetes_pod_operator.py:docstring of airflow.contrib.operators.kubernetes_pod_operator.KubernetesPodOperator.execute:1: WARNING: duplicate object description of airflow.contrib.operators.kubernetes_pod_operator.KubernetesPodOperator.execute, other instance in /workspace/docs/code.rst, use :noindex: for one of them
```

**Comment by feng-tao** [comment]
@mik-laj not sure what you mean by "doc change". Are you saying that the docstring in the code file is not considered a code change?

**Comment by mik-laj** [comment]
@feng-tao Yes. Any change in the documentation, no matter what form it takes, does not require a JIRA ticket.
The changes I have made do not change the code. The code is processed by the computer. The documentation that is read by man is changed.

I have prepared a report for you that presents this principle:
````
993779f93df172f5f774f1ea2634c55e9c1212c4 [AIRFLOW-XXX] Reduction of the number of warnings in the documentation (#4585)}
	airflow/contrib/hooks/aws_glue_catalog_hook.py
	airflow/contrib/hooks/azure_container_instance_hook.py
	airflow/contrib/hooks/azure_container_volume_hook.py
	airflow/contrib/hooks/cassandra_hook.py
	airflow/contrib/hooks/ftp_hook.py
	airflow/contrib/hooks/gcp_bigtable_hook.py
	airflow/contrib/hooks/gcp_compute_hook.py
	airflow/contrib/hooks/gcp_container_hook.py
	airflow/contrib/hooks/gcp_mlengine_hook.py
	airflow/contrib/hooks/gcp_pubsub_hook.py
	airflow/contrib/hooks/gcp_spanner_hook.py
	airflow/contrib/hooks/gcs_hook.py
	airflow/contrib/hooks/mongo_hook.py
	airflow/contrib/hooks/sftp_hook.py
	airflow/contrib/hooks/spark_jdbc_hook.py
	airflow/contrib/hooks/spark_sql_hook.py
	airflow/contrib/hooks/wasb_hook.py
	airflow/contrib/operators/adls_to_gcs.py
	airflow/contrib/operators/azure_container_instances_operator.py
	airflow/contrib/operators/bigquery_to_bigquery.py
	airflow/contrib/operators/databricks_operator.py
	airflow/contrib/operators/gcs_operator.py
	airflow/contrib/operators/gcs_to_gcs.py
	airflow/contrib/operators/jenkins_job_trigger_operator.py
	airflow/contrib/operators/mlengine_operator.py
	airflow/contrib/operators/qubole_check_operator.py
	airflow/contrib/operators/sqoop_operator.py
	airflow/contrib/operators/wasb_delete_blob_operator.py
	airflow/contrib/sensors/python_sensor.py
	airflow/contrib/sensors/weekday_sensor.py
	airflow/hooks/dbapi_hook.py
	airflow/hooks/http_hook.py
	airflow/hooks/mysql_hook.py
	airflow/hooks/oracle_hook.py
	airflow/models/__init__.py
	airflow/operators/s3_file_transform_operator.py
	airflow/task/task_runner/__init__.py
	airflow/task/task_runner/base_task_runner.py
	airflow/utils/dag_processing.py
	airflow/utils/helpers.py
	docs/conf.py
c69619e9cf823cdc4abd3874a296ce22b16349d3 [AIRFLOW-XXX] Fix spark submit hook KeyError (#4578)}
	airflow/contrib/hooks/spark_submit_hook.py
95d6c4294a12b38d46ae19a70e404b4521cc1d21 [AIRFLOW-XXX] Fix doc string of DataprocClusterDeleteOperator (#4565)}
	airflow/contrib/operators/dataproc_operator.py
ac25322da22f7c59cbe0d65f4b737c1decc10872 [AIRFLOW-XXX] Fix typo (#4564)}
	airflow/contrib/hooks/bigquery_hook.py
	airflow/contrib/hooks/salesforce_hook.py
	airflow/contrib/operators/dataflow_operator.py
	airflow/contrib/operators/mlengine_operator.py
	airflow/contrib/operators/oracle_to_azure_data_lake_transfer.py
	airflow/contrib/operators/qubole_operator.py
	airflow/contrib/operators/s3_to_sftp_operator.py
	airflow/contrib/operators/sftp_operator.py
	airflow/contrib/operators/sftp_to_s3_operator.py
	airflow/contrib/operators/ssh_operator.py
	airflow/contrib/plugins/metastore_browser/main.py
	airflow/contrib/utils/weekday.py
	airflow/operators/s3_file_transform_operator.py
	airflow/sensors/base_sensor_operator.py
	airflow/ti_deps/dep_context.py
	airflow/utils/sqlalchemy.py
	tests/ti_deps/deps/fake_models.py
ad77b17a7532a5483396b2109f5bd10a3d31cc5d [AIRFLOW-XXX] Correct Typo in sensor's exception (#4545)}
	airflow/sensors/external_task_sensor.py
f9ca8e4cb99d9084b0b4e827f2772e6c5adf9746 [AIRFLOW-XXX] Remove `of to` typo. (#4542)}
	airflow/contrib/operators/kubernetes_pod_operator.py
dba13d9bf09dbf273bc8e2e990002cc00ed57714 [AIRFLOW-XXX] Fix misspelling of Vertica (#4509)}
	airflow/contrib/operators/vertica_to_hive.py
b450065dc2422408a6cb828ade69804956174ca4 [AIRFLOW-XXX] Fix a flake8 error to unblock CI (#4453)}
	airflow/contrib/operators/emr_create_job_flow_operator.py
97c761ddabcee08deb1f774f637a1bafe22a3de6 [AIRFLOW-XXX] Fix WeekDay Sensor Example (#4431)}
	airflow/contrib/sensors/weekday_sensor.py
51d0aca83bc9b5c4cd073466fa7df2a43bd871bd [AIRFLOW-XXX] Fix Flake8 error}
	tests/contrib/operators/test_bigquery_operator.py
ecc77c2bf706a61d1dd77b1a95a00b74f29f74e3 [AIRFLOW-XXX] Fix Flake8 error (#4422)}
	airflow/contrib/operators/mysql_to_gcs.py
4b99adf1369f5f56f32939139363af84c7f2671f [Airflow-XXX] - fix missing type in docstring (#4403)}
	airflow/contrib/operators/mysql_to_gcs.py
f8903fb22c6e6e468b25fc1ade07602a5b055b59 [AIRFLOW-XXX] Fix inconsistent comment in example_python_operator.py (#4337)}
	airflow/example_dags/example_python_operator.py
09a47c1ec537e09c7340678d9dfb94fd0e82cdf3 [AIRFLOW-XXX] Fix incorrect parameter in SFTPOperator example (#4344)}
	airflow/contrib/operators/sftp_operator.py
437196917f7e084f0a3ddd532dc19e9bc47a66f1 [AIRFLOW-XXX] Fix Minor issues with Azure Cosmos Operator (#4289)}
	airflow/contrib/example_dags/example_azure_cosmosdb_sensor.py
	airflow/contrib/example_dags/example_cosmosdb_sensor.py
	airflow/contrib/operators/azure_cosmos_insertdocument_operator.py
	airflow/contrib/operators/azure_cosmos_operator.py
	tests/contrib/operators/test_azure_cosmos_insertdocument_operator.py
4cde579d12054e06f34351469f7d70863950729f [AIRFLOW-XXX] GCP operators documentation clarifications (#4273)}
	airflow/contrib/hooks/gcp_sql_hook.py
	tests/contrib/operators/test_gcp_sql_operator.py
22bd7c6da561ca9a6076cecb878caebcadd779f4 [AIRFLOW-XXX] Fix display of SageMaker operators/hook docs (#4263)}
	airflow/contrib/hooks/sagemaker_hook.py
	airflow/contrib/operators/sagemaker_endpoint_config_operator.py
	airflow/contrib/operators/sagemaker_endpoint_operator.py
	airflow/contrib/operators/sagemaker_model_operator.py
	airflow/contrib/operators/sagemaker_training_operator.py
	airflow/contrib/operators/sagemaker_transform_operator.py
	airflow/contrib/operators/sagemaker_tuning_operator.py
	docs/conf.py
77e233e525cab747f66e7a04a13ce8d233146335 [AIRFLOW-XXX] Add missing GCP operators to Docs (#4260)}
	airflow/contrib/operators/mysql_to_gcs.py
d8c0b639743009dcfdc0bbe513319e0ed8e7de6e [AIRFLOW-XXX] Remove unnecessary "# noqa" in airflow/bin/cli.py (#4223)}
	airflow/bin/cli.py
ef633e6f2aaaab7acc5d8fddb9c943c5938b7be1 [AIRFLOW-XXX] Fix flake8 failure from #4184}
	airflow/operators/bash_operator.py
43d037f71f8e2a4b98e925fd30c676073a165ab7 [AIRFLOW-XXX] Speed up RBAC view tests (#4162)}
	tests/www_rbac/test_views.py
b1d9111fc6d8305e4720111029a9212083e717d7 [AIRFLOW-XXX] Pin Flask App Builder to Fix CI (#4150)}
	setup.py
dc0eb58e97178b050b79584f18d8b9bd2c3dea5f [AIRFLOW-XXX] Fix flake8 errors from #4144}
	tests/operators/test_http_operator.py
dab0c78437bf81c656fd6195420e37fe774ae377 [AIRFLOW-XXX] Use mocking in SimpleHttpOperator tests (#4144)}
	tests/operators/test_http_operator.py
a5dbdd390e24d11a823668ef6abdba124cb548bb [AIRFLOW-XXX] Fix Docstrings in Hooks, Sensors & Operators (#4137)}
	airflow/contrib/hooks/spark_submit_hook.py
	airflow/contrib/hooks/sqoop_hook.py
	airflow/contrib/operators/bigquery_operator.py
	airflow/contrib/operators/dataproc_operator.py
	airflow/contrib/operators/gcs_to_s3.py
	airflow/contrib/operators/pubsub_operator.py
	airflow/contrib/operators/s3_delete_objects_operator.py
	airflow/contrib/operators/s3_list_operator.py
	airflow/contrib/operators/s3_to_gcs_operator.py
	airflow/contrib/operators/spark_jdbc_operator.py
	airflow/contrib/sensors/bigquery_sensor.py
	airflow/contrib/sensors/sftp_sensor.py
	airflow/operators/http_operator.py
	airflow/operators/python_operator.py
	airflow/operators/redshift_to_s3_operator.py
	airflow/operators/s3_to_hive_operator.py
	airflow/operators/s3_to_redshift_operator.py
	airflow/sensors/s3_key_sensor.py
	airflow/sensors/s3_prefix_sensor.py
	airflow/sensors/sql_sensor.py
45f784a3574602f72bc835a132b042a647efe994 [AIRFLOW-XXX] BigQuery Hook - Minor Refactoring (#4066)}
	airflow/contrib/hooks/bigquery_hook.py
33469c23ed2714cf467af869e3b69e681f38da03 [AIRFLOW-XXX] Fix GCS Operator docstrings (#4054)}
	airflow/contrib/operators/gcs_to_gcs.py
f9f7afda8c99e1f840ebfa349956ba86b7401e5b [AIRFLOW-XXX] Add missing docstring in GCS Hook & Operator (#4053)}
	airflow/contrib/hooks/gcs_hook.py
f840fca13120fa94179d49825b463cd61603db86 [AIRFLOW-XXX] Fix BashOperator Docstring (#4052)}
	airflow/operators/bash_operator.py
ac87e15201d5a662639a2c79bb74149fe138dc70 [AIRFLOW-XXX] Fix Typo in SFTPOperator docstring (#4016)}
	airflow/contrib/operators/sftp_operator.py
b85210d8596c1e05fbf3da2941aaca8170ae871e [AIRFLOW-XXX] Fix  airflow.models.DAG docstring mistake}
	airflow/models.py
9f61f5b9fe3136f077e0dced9729f8284ddfcc88  [AIRFLOW-XXX] Fixing the issue in Documentation (#3998)}
	airflow/contrib/operators/dataflow_operator.py
c377b4b5db607011c6c08607c6b8bfa3cfe5c50e [AIRFLOW-XXX] Fix undocumented params in S3_hook}
	airflow/hooks/S3_hook.py
7a6f4b013335d42eb7005f49ff202da147b90af1 [AIRFLOW-XXX] Don't spam test logs with "bad cron expression" messages (#3973)}
	tests/jobs.py
f101352056d5115594b8e010bb7bdcc2e4a44633 [AIRFLOW-XXX] Speed up DagBagTest cases (#3974)}
	tests/models.py
6a7f3887460c91bc4da90fe8d31836a225324a1a [AIRFLOW-XXX] Fix PythonVirtualenvOperator tests (#3968)}
	tests/operators/test_virtualenv_operator.py
9238bce3174c1c7dc660aa61275d54632f507719 [AIRFLOW-XXX] Fix SlackWebhookOperator execute method comment (#3963)}
	airflow/contrib/operators/slack_webhook_operator.py
f4f8027cbf61ce2ed6a9989facf6c99dffb12f66 [AIRFLOW-XXX] Pin Click to 6.7 to Fix CI (#3962)}
	setup.py
5355909b5f4ef0366e38f21141db5c95baf443ad [AIRFLOW-XXX] Fix SlackWebhookOperator docs (#3915)}
	airflow/contrib/operators/slack_webhook_operator.py
eb607e3217a406f4b14d9c0d8fac9db8c1d2b7e7 [AIRFLOW-XXX] Fix typo in docstring of gcs_to_bq (#3833)}
	airflow/contrib/operators/gcs_to_bq.py
ba27fcad69fa914cc55c530f1716b676974c7bd5 [AIRFLOW-XXX] Fix Docstrings for Operators (#3820)}
	airflow/contrib/hooks/gcp_container_hook.py
	airflow/contrib/operators/awsbatch_operator.py
	airflow/contrib/operators/bigquery_check_operator.py
	airflow/contrib/operators/bigquery_operator.py
	airflow/contrib/operators/dataflow_operator.py
	airflow/contrib/operators/dataproc_operator.py
	airflow/contrib/operators/ecs_operator.py
	airflow/models.py
	airflow/operators/s3_file_transform_operator.py
	airflow/sensors/s3_key_sensor.py
	airflow/sensors/s3_prefix_sensor.py
27309b13f17402eaa61d4e4fede8785effa8bbb7 [AIRFLOW-XXX] Fix minor typo (#3776)}
	airflow/executors/base_executor.py
ccd2b88a4201b8d1d4a71f24934135c212ce1a54 [AIRFLOW-XXX] Fixing the issue in Documentation (#3756)}
	airflow/contrib/operators/dataflow_operator.py
9d516c7134eb22a3c2fc63cf96626ef6e8b247f2 [AIRFLOW-XXX] Make pip install commands consistent (#3752)}
	airflow/contrib/utils/sendgrid.py
	airflow/example_dags/example_kubernetes_operator.py
1e991b1b5071c8109e735902d62baaf1659a54d6 [AIRFLOW-XXX] Update BaseOperator documentation on priority_weight (#3727)}
	airflow/models.py
7c19ed02bc30f71fd73e38234b3b210072274f3b [AIRFLOW-1729][AIRFLOW-XXX] Remove extra debug log at info level}
	airflow/models.py
a8dd6d86d512af2dcaf77270b18f2f6366bff5e6 [AIRFLOW-1163][AIRFLOW-XXX] Add support for x-forwarded-* headers}
	airflow/www/app.py
	airflow/www_rbac/app.py
0ce068e0867f9105f3a1a92d389bc41e6da85991 [AIRFLOW-XXX] Fix typo in http_operator.py}
	airflow/operators/http_operator.py
1c8e5808d4b7b5ebe932944921e38ccbd208d1b2 [AIRFLOW-XXX] Typo fix}
	airflow/contrib/operators/mongo_to_s3.py
ba126926d7d8a3d8d5bae548c12b5ce4b2a9fbd8 [AIRFLOW-XXX] Add timeout units (seconds)}
	airflow/contrib/operators/ssh_operator.py
375ed75ff1ba1abf16e18ae29b62320bebbf860f [AIRFLOW-XXX] Typo node to nodes}
	airflow/models.py
5e4d7d8d7da250cacc15d8a2247098f0b42a3def [AIRFLOW-XXX] Pin sqlalchemy dependency}
	setup.py
81558f3d094edd6bc8aac4ab0528f561c65e2b3e [AIRFLOW-XXX] Fix failing PubSub tests on Python3}
	tests/contrib/hooks/test_gcp_pubsub_hook.py
	tests/contrib/operators/test_pubsub_operator.py
51180d370cae066dbb8625442d1723b63b3f60e7 [AIRFLOW-XXX] Upgrade to python 3.5 and disable dask tests}
	tests/executors/dask_executor.py
f9dcab0fa247b50e6655500f187670e908954149 [AIRFLOW-XXX] Fix typo in comment}
	airflow/models.py
a8d486de337779e6319b90215fd8ea7faa5f5913 [AIRFLOW-XXX] Bumping Airflow 1.10.0dev0+incubating version}
	airflow/version.py
````

I hope that my explanation is enough for you.

**Pull request merged**